src/CMakeLists.txt

```diff
@@ -5,7 +5,10 @@
 # These apps may also require the building of openglbuilder from utils
```

Comment:
Could you fix the following typo `utils` to `libutils`?

Reply:
.. and `openglbuilder` to `oglapphelpers`.

Reply:
Fixed in next commit